### PR TITLE
fix: make launchpad link open default browser

### PR DESCRIPTION
**Pull request by elevatebart**
- Closes UNIFY-1000

### User facing changelog

Instead of the link below (Learn more.) opening in a new electron window, it now opens in the default browser.

Screen Shot 2022-02-28 at 10 43 19 AM

- [x] Have tests been added/updated?

### CODE REVIEW

File: packages/launchpad/src/components/code/FileRow.vue [hunk 1]
```diff
@@ -97,7 +98,7 @@ const props = defineProps<{
   description?: string
 }>()
 
-const openDocs = () => window.open('https://on.cypress.io/guides/configuration')
```
Comment (tbiethman) [inline packages/launchpad/src/components/code/FileRow.vue:100]: I think we could just set the `href` prop on the Button and not have to worry about the right mutation here: https://github.com/cypress-io/cypress/blob/7c244f6b245a71685c019457bc76abdd626b123a/packages/frontend-shared/src/components/Button.vue#L122

Comment (elevatebart) [reply packages/launchpad/src/components/code/FileRow.vue:100]: It will not be cool with router links...

Comment (tbiethman) [reply packages/launchpad/src/components/code/FileRow.vue:100]: That's what the `internalLink`/`to` props are for 😛  lots of props on Button apparently

https://github.com/cypress-io/cypress/blob/7c244f6b245a71685c019457bc76abdd626b123a/packages/frontend-shared/src/components/Button.vue#L124

Comment (elevatebart) [reply packages/launchpad/src/components/code/FileRow.vue:100]: ... awwwww no! that's horrible.

I can do that... even if I am dying inside.

File: packages/launchpad/cypress/e2e/project-setup.cy.ts [hunk 2]
```diff
@@ -815,4 +815,22 @@ describe('Launchpad: Setup Project', () => {
...
+
+      cy.get('[data-cy-testingtype="component"]').click()
+      cy.get('[data-testid="select-framework"]').click()
+      cy.findByText('Nuxt.js (v2)').click()
+      cy.findByText('Next Step').click()
+      fakeInstalledDeps()
+      cy.findByRole('button', { name: 'Continue' }).click()
+      cy.intercept('POST', 'mutation-ExternalLink_OpenExternal').as('OpenExternal')
+      cy.findByText('Learn more.').click()
+      cy.wait('@OpenExternal')
```
Comment (lmiller1990) [inline packages/launchpad/cypress/e2e/project-setup.cy.ts:833]: Should we assert the correct arg was included in the request?

File: packages/launchpad/src/components/code/FileRow.vue (hunk 1 above)
Comment (lmiller1990) [reply packages/launchpad/src/components/code/FileRow.vue:100]: I hate that `<Button>` takes a `href` but this seems like the way to go. We could fix this eventually all in one shot. Do you plan to do this @elevatebart ?

File: packages/launchpad/cypress/e2e/project-setup.cy.ts (hunk 2 above)
Comment (lmiller1990) [reply packages/launchpad/cypress/e2e/project-setup.cy.ts:833]: Edit: probably irrelevant if you make the change mentioned by @tbiethman.

**Review by lmiller1990 — COMMENTED** [review]
Same comment as Tyler! What do you think?

File: packages/launchpad/src/components/code/FileRow.vue (hunk 1 above)
Comment (elevatebart) [reply packages/launchpad/src/components/code/FileRow.vue:100]: I just did ;)

**Comment by elevatebart** [comment]
@tbiethman @lmiller1990 Updated in the use of href in cad49a1

File: packages/launchpad/cypress/e2e/project-setup.cy.ts (hunk 2 above)
Comment (elevatebart) [reply packages/launchpad/cypress/e2e/project-setup.cy.ts:833]: Added in commit 15206c91e44ea883215b0a5c4d0f5f736734ff7e

**Review by lmiller1990 — APPROVED** [review]

**Review by elevatebart — COMMENTED** [review]
responded

**Comment by lmiller1990** [comment]
Gonna merge since the request of Tyler is already done :heavy_check_mark:

**Pull request merged**